### PR TITLE
[DO NOT MERGE] Content and go live branch for pre-need form

### PR DESCRIPTION
**Pull request by peggygannon**
This PR is an update to the pre-need static page in time for the online form launch. 
Pages that have updates: 
-[ ]top nav
-[ ]burials-and-memorials/pre-need/ (addition of Green apply button)

### CODE REVIEW

**Comment by peggygannon** [comment]
@bethpotts PR for Pre-need page update is here: https://github.com/department-of-veterans-affairs/vets-website/pull/6841.

File: content/includes/veteran-navigation.html [hunk 1]
```diff
@@ -224,6 +224,7 @@
                   <li><a href="/burials-and-memorials/" onClick="reportHeaderNav('Explore Benefits->Burials Memorials');">Burials &amp; memorials overview</a></li>
                   <li><a href="/burials-and-memorials/eligibility/" onClick="reportHeaderNav('Explore Benefits->Burials Memorials');">Eligibility</a></li>
                   <li><a href="/burials-and-memorials/burial-planning/" onClick="reportHeaderNav('Explore Benefits->Burials Memorials');">Plan a burial</a></li>
+                  <li><a href="/burials-and-memorials/pre-need/" onClick="reportHeaderNav('Explore Benefits->Burials Memorials');">Plan ahead for your own burial by <br>applying for pre-need eligibility</a></li>
```
Comment (jbalboni) [inline content/includes/veteran-navigation.html]: That you need to put a `<br>` in here is a bug, I think. Can you open a ticket about text not wrapping in the navigation menu?

**Review by jbalboni — APPROVED** [review]
LGTM

**Comment by peggygannon** [comment]
@jbalboni This PR is ready to merge into the pre-need form/application. 

I tried to resolve the 1 conflict, but the text seems to be the same and I'm not sure what the conflict is.

**Comment by jbalboni** [comment]
@peggygannon I merged master in, can you check again to make sure your changes look correct?

**Comment by peggygannon** [comment]
@jbalboni I just gave this PR a review and need help with a few things:
--in the top nav file, I removed a break tag, but then the text didn't wrap. I re-instated the break tag. Is that OK to have in the top nav? 

--on the /burials-and-memorials/pre-need/ page, there is a link in the left nav that goes to the pre-need application. We need to remove this link, since the /pre-need/ page will now have a green apply button.  I couldn't find this link/text in the files, though, to remove. I'm wondering where this link lives? here's a screenshot of the page:

**Comment by peggygannon** [comment]
@jbalboni Thanks, Jeff. I don't need to update anything else with this PR, so you can go in and fix this. 

thank you!

**Comment by jbalboni** [comment]
@peggygannon Can you make sure the changes that show up now are what you expect?

**Comment by peggygannon** [comment]
@jbalboni Hi, Jeff. All looks good. Thank you!

**Pull request merged**